worker-workflow/src/main/resources/workflow-control.js

```diff
@@ -371,17 +371,29 @@ function isEmptyMap(mapValue) {
     return jsonString === '{}';
 }
 
+function searchFieldsForValue(fieldValues, value) {
+    if (fieldValue.isStringValue() && fieldValue.getStringValue() === value) {
```

Comment:
Parameter is `fieldValues` which doesn't seem to be used in function, but `fieldValue` is used and is not in the parameter list.  Also rename the function to something more in line with what it now does.

Reply:
JS scope must have allowed my tests to pass with these mistakes. Fixes pushed up now.